plugins/auth-backend/api-report.md

```diff
@@ -59,9 +59,17 @@ export type Auth0ProviderOptions = {
   };
 };
 
+// @public
+export type AuthContext = {
```

Comment:
Feeling rename here would still be good, for example `AuthResolverContext`, I think it's a bit too generic as it is

Reply:
I renamed it from `AuthResolverContext` to `AuthContext` as it is used for the resolver and for the handler - it just sounded to be for the resolver only. But I wasn't aware of your [previous comment](https://github.com/backstage/backstage/pull/8396#discussion_r770926702), sorry for that 😉

Just to bring another name in the discussion, what do you think about `AuthProviderContext`?

Reply:
After some discussion: Go with `AuthResolverContext`.

Reply:
Okay for me 👍

Reply:
Thanks for the clarification @Rugvip I just pushed the renaming 😉